doc-content/jbpm-docs/src/main/asciidoc/BPMN2/bpmn-intermediate-events-ref.adoc

```diff
@@ -1,7 +1,14 @@
 [id='bpmn-intermediate-events-ref_{context}']
 
 = Intermediate events
```

Comment:
@hmanwani-rh , can you check with your SME and and Cristiano if these Intermediate events updates or any other updates in this PR also need to be made to Kogito docs? Here for example: https://github.com/kiegroup/kie-docs/blob/master-kogito/doc-content/kogito-docs/src/main/asciidoc/bpmn/chap-kogito-developing-process-services.adoc#intermediate-events-supported-in-product  If they do, would you mind making the needed updates? Thanks.

Reply:
@hasys, can you please confirm if this change can also be applied to kogito docs?

Reply:
Hi @sterobin, @hmanwani-rh, Kogito Jira issue is not implemented yet, see https://issues.redhat.com/browse/KOGITO-4149